addons/account/wizard/account_invoice_send.py

```diff
@@ -46,6 +54,26 @@ def _compute_composition_mode(self):
         for wizard in self:
             wizard.composer_id.composition_mode = 'comment' if len(wizard.invoice_ids) == 1 else 'mass_mail'
 
+    @api.onchange('invoice_ids')
+    def _compute_move_types(self):
+        for wizard in self:
+            move_types = False
+
+            if len(wizard.invoice_ids) > 1:
+                moves = self.env['account.move'].search([
+                    ('id', 'in', self.env.context.get('active_ids')),
+                ])
```

Comment:
Why not a browse?

Reply:
It does make more sense indeed. I've changed that and also avoided a traceback when clicking 'send & print' with a move that has no partner_id.